src/main/java/org/apache/maven/resolver/internal/ant/tasks/Deploy.java

```diff
@@ -46,13 +113,71 @@ protected void validate() {
         }
     }
 
+    /**
+     * Allows ant to add a remote repository to which artifacts will be deployed.
```

Comment:
to which artifacts will be deployed. -> which artifacts will be deployed to.

(I don't care what grammar school teacher say about ending sentences with prepositions)

Reply:
Yes, I agree that this is cleared, changed it.